Feature-IOPS-1874 update to Profile-England-Task

KazeemHamzat opened the pull request:
-Adding the missing CodeSystem and ValueSet to Profile-England-Task
-Update OperationOutcome to include meta.lastUpdated element
-Update Profile-England-ServiceRequest to reflect ValuseSet-England-ServiceRequestCodes
-Update ID and URL for ValuseSet-England-ServiceRequestCodes

KazeemHamzat committed:
Update to ServiceRequest & OperationOutcome

Update to ServiceRequest & OperationOutcome

KazeemHamzat committed:
Update; Profile-England-Task

Adding the missing CodeSystem and ValueSet to Profile-England-Task
Update OperationOutcome to include meta.lastUpdated element
-Update Profile-England-ServiceRequest to reflect ValuseSet-England-ServiceRequestCodes
-Update ID and URL for ValuseSet-England-ServiceRequestCodes

KazeemHamzat committed:
Update Valueset-England-WorkflowCode.xml

Update to the Canonical URL

digitalhealthmartin commented on structuredefinitions/England-OperationOutcome.xml:
```diff
@@ -27,17 +27,19 @@
     </element>
     <element id="OperationOutcome.meta">
       <path value="OperationOutcome.meta" />
+      <min value="1" />
```
Why min changed to 1?

KazeemHamzat replied:
That was the original value in [NHSDigital-OperationOutcome](https://simplifier.net/NHSDigital/NHSDigital-OperationOutcome/~json).

digitalhealthmartin commented on codesystems/CodeSystem-England-WorkflowCategoryCodes.xml:
```diff
@@ -0,0 +1,31 @@
+            <rank value="1" />
+        </telecom>
+    </contact>
+    <description value="workflow codes for use with (sub-)Tasks" />
```
w -> W

digitalhealthmartin commented on structuredefinitions/England-OperationOutcome.xml:
```diff
@@ -27,17 +27,19 @@
+      <path value="OperationOutcome.meta.lastUpdated" />
+      <min value="1" />
+    </element>
     <element id="OperationOutcome.extension">
```
Why min changed to 1?

KazeemHamzat replied:
That was the original value in [NHSDigital-OperationOutcome](https://simplifier.net/NHSDigital/NHSDigital-OperationOutcome/~json).

digitalhealthmartin commented on valuesets/Valueset-England-ActionCode.xml:
Valueset -> ValueSet

KazeemHamzat replied:
Changes will be done in branch: Feature-[IOPS-1856](https://nhsd-jira.digital.nhs.uk/browse/IOPS-1856)-Task-Examples
This file will be deleted from this branch.

digitalhealthmartin commented on valuesets/Valueset-England-ServiceRequestCodes.xml:
Valueset -> ValueSet

digitalhealthmartin commented on valuesets/Valueset-England-ServiceRequestCodes.xml:
```diff
@@ -1,6 +1,6 @@
 <ValueSet xmlns="http://hl7.org/fhir">
-    <id value="England-ServiceRequestCategoryCodes" />
-    <url value="https://fhir.nhs.uk/ValueSet/England-ServiceRequestCategoryCodes" />
+    <id value="England-ServiceRequestCodes" />
```
Why id/url changed from ServiceRequestCategoryCodes to ServiceRequestCodes? 'Category' has been removed

KazeemHamzat replied:
ServiceRequest codes is the slice name for the element. <sliceName value="ServiceRequestCode" />
Changing it will align properly with the Slice name (as previous done in [NHSDigital-ServiceRequest](https://simplifier.net/NHSDigital/NHSDigitalServiceRequest/~xml). 
Also, it will properly differentiate it from a valueset with a similar name: <valueSet value="https://fhir.nhs.uk/ValueSet/England-ServiceRequestCategory"

digitalhealthmartin commented on valuesets/Valueset-England-TaskCode.xml:
Valueset -> ValueSet

KazeemHamzat replied:
Changes will be done in branch: Feature-IOPS-1856-Task-Examples
This file will be deleted from this branch.

digitalhealthmartin commented on valuesets/Valueset-England-WorkflowCode.xml:
Valueset -> ValueSet

KazeemHamzat replied:
GitHub doesnt seem to recognise that as a change. ValueSet and Valueset are the same for it.
Changed locally but its not reflecting on GitHub.

KazeemHamzat committed:
Update to Profiles

-Removing extra quotes from OperationOutcome and Immunization.
-Fixing PR Issues in ValueSets
